File: code/003/python/main.py
"""From Python to Go: Python: 003 - imports and function"""

# Import part
import os


# Functions
def get_username() -> str:
    """Helper function to get username"""
    return os.getenv("AUTOMATION_USERNAME", None)


def get_password() -> str:
    """Helper function to get password"""
    return os.getenv("AUTOMATION_PASSWORD", None)


# Execution
username = get_username()
password = get_password()

print(f"{username=}, {password=}")

File: code/003/go/main.go
/*From Python to Go: Go: 003 - imports and function*/
package main

// Import part
import (
	"fmt"
	"os"
)

// Aux functions
func getUsername() string {
	/* Helper function to get username */
	return os.Getenv("AUTOMATION_USERNAME")
}

func getPassword() string {
	/* Helper function to get password */
	return os.Getenv("AUTOMATION_PASSWORD")
}

// Main
func main() {
	// get variables
	username := getUsername()
	password := getPassword()

	fmt.Printf("username=%v, password=%v\n", username, password)
}
